scripts/pythondistdeps.py

```diff
@@ -142,9 +210,23 @@ def convert(name, operator, version_id):
 
 
 def normalize_name(name):
```

Comment:
This function is now completely unused and can be removed.

Reply:
It can't, and I wish it could be -- https://src.fedoraproject.org/rpms/python-rpm-generators/blob/master/f/pythonbundles.py#_41

Reply:
Ah, right. In that case, yeah, a comment would be great.

Reply:
Happy to adapt that script when we integrate this.

Reply:
@s-t-e-v-e-n-k Thinking more on this, since we have to keep this function, and since the class/static methods you currently use are used inside 2 different classes anyway, how about implementing the functionality as normal global functions? That way they could be used outside this script easily, and we would not need this wrapper.

Reply:
resolved